### PR TITLE
chore(translations): Clean up unused translations

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
We have some legacy translations that are not used anymore but still included in the language bundles which are bloating things.

Closes: AVO-322

## Description

This PR adds a script to remove any key that are not in the `en.json` file from any of the other language files.

It removes some translations from `en.json` that are no longer used.

Runs the script.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/src/locales/sv.json [hunk 1]
```diff
@@ -134,7 +132,6 @@
     },
     "estimated_time_slicer_average": {
       "title": "Data är för närvarande uppskattad",
-      "pill": "Otillgänglig",
```
Comment (VIKTORVAV99) [inline web/src/locales/sv.json:137]: I wonder if we should just add this key to the en file instead?

File: web/src/locales/pt-PT.json [hunk 2]
```diff
@@ -980,8 +964,7 @@
       "zoneName": "Índia Nordeste"
     },
     "IN-NO": {
-      "countryName": "Índia",
-      "+oneName": "Índia Norte"
```
Comment (VIKTORVAV99) [inline web/src/locales/pt-PT.json:984]: I'll clean up this one.

File: web/src/locales/pt-PT.json [hunk 3]
```diff
@@ -132,7 +132,6 @@
     },
     "estimated_time_slicer_average": {
       "title": "Os dados atuais são estimados",
-      "pill": "Indisponível",
```
Comment (VIKTORVAV99) [inline web/src/locales/pt-PT.json:135]: Same here as for the Swedish one

File: web/src/locales/sv.json (hunk 1 above)
Comment (tonypls) [reply web/src/locales/sv.json:137]: I think it's okay to remove for now. We should re-add this with translations from "Preliminary" in the future, English also needs this for the badges in AVO-282. I'll add some context there

**Comment by VIKTORVAV99** [comment]
> LGTM, it's a little confusing now we have two en.json, nice work all the same!

Yeah and I think we should consider moving it directly to the API project but we can discuss that on a later point if we need to actually move it.

**Pull request merged**